app/views/search/index.html.erb

```diff
@@ -38,7 +38,7 @@ See docs/COPYRIGHT.rdoc for more details.
 <global-search-tabs></global-search-tabs>
 
 <% if params[:work_packages].present? %>
-  <global-search-work-packages></global-search-work-packages>
+  <global-search-work-packages-entry></global-search-work-packages-entry>
```

Comment:
You can't, because what we're doing from Rails is rendering a custom element such as `global-search-work-packages`, and then creating a component from that selector. You don't have structural directives such as `ng-container, *ngIf` etc. in the Rails template, it will not be compiled by Angular, and thus an entry component is required.

Reply:
Oh, I see. Tx.